openapi/data_repository_service.swagger.yaml

```diff
@@ -396,3 +405,15 @@ definitions:
       license:
         type: object
         description: License information for the exposed API
+  BundleObjects:
+    type: object
+    properties:
+      name:
+        type: string
+      id:
+        type: string
```

Comment:
please add a description, clarifying:
1. that this is the ID of a DRS object
2. whether it can also be the ID of a DRS bundle (and if so, how I know which is which)

Reply:
Description added. I have added `type` to the BundleObject to help the user make this distinction.